src/viking.js

```diff
@@ -1,11 +1,53 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(healthArg, strengthArg) {
+    this.health = healthArg;
+    this.strength = strengthArg;
+  }
 
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+  }
+}
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(nameArg, healthArg, strengthArg) {
+    super(healthArg, strengthArg);
+    this.name = nameArg;
+  }
+  receiveDamage(damage) {
+    this.health = this - damage;
```

Comment:
Also, here you just forgot to put "this.health" after the equal sign, you just wrote "this".